core/check.py

```diff
@@ -151,7 +151,10 @@ async def wrapper(*args, **kwargs):
                 return await func(*args, **kwargs)
 
             if reply and isinstance(args[1], commands.Context):
-                await args[1].message.reply(tr("version", "reply"))
+                ctx = None
+                await args[1].message.reply(
+                    _(ctx, "This command is not available in this version")
+                )
```

Comment:
You can see that when we're inside of this `if` statement, the `args[1]` _is_ the Context we're using in translation. So, we can write
```py
await args[1].message.reply(
    _(args[1], "This command is not available in this version")
)
```

Reply:
resolved in b4e6d7d0a064707ca1c52d507e9d138a2309a43e